release/ray_release/tests/test_retry.py

```diff
@@ -0,0 +1,39 @@
+import sys
```

Comment:
We don't really need this test, right? Assuming that the retry library just works

Reply:
done

Reply:
done

Reply:
done